lib/linter.js

```diff
@@ -174,6 +178,22 @@ var runChecks = function (ast, fullPath, lines, config) {
     return results;
 };
 
+var mergeLinters = function (defaultLinters, config) {
+    if (!config.linters) {
+        return defaultLinters;
+    }
+
+    if (!(config.linters instanceof Array)) {
+        throw new Error('linters should be an array of file paths');
+    }
+
+    return defaultLinters.concat(
+        config.linters.map(
+            function (linterPath) {
+                return require(linterPath);
```

Comment:
this assumes a string. I'd like to see support for already-required modules (eg. Objects) here. 

```js
linters: [
  require('./lib/my-fancy-linter')(someConfig);
]
```

That'd negate the need for another 'linterConfig' section for configuring dynamic linters at load-time. I know we don't really do that now, but something like the proposed browser prefix linter could potentially use multiple source, and then we'd be mixing load-config with linter-config, or stuck creating another section. that smells of Grunt type config, and I'd like to avoid that. instead, following on the path of modules like babel, supporting the already-required modules.

(it's early here, let me know if that doesn't make sense :))